index.js

```diff
@@ -85,14 +85,41 @@ export async function runASTAnalysisOnFile(pathToFile, options = {}) {
 
 export const CONSTANTS = {
   Warnings: Object.freeze({
-    parsingError: "ast-error",
-    unsafeImport: "unsafe-import",
-    unsafeRegex: "unsafe-regex",
-    unsafeStmt: "unsafe-stmt",
-    unsafeAssign: "unsafe-assign",
-    encodedLiteral: "encoded-literal",
-    shortIdentifiers: "short-identifiers",
-    suspiciousLiteral: "suspicious-literal",
-    obfuscatedCode: "obfuscated-code"
+    parsingError: {
+      code: "ast-error",
+      i18n: "ast_error"
```

Comment:
`warnings.ast_error`. You need the full i18n path here (same for all the i18n properties below).